03-auto_tick.xsh

```diff
@@ -174,6 +174,8 @@ def run(feedstock=None, protocol='ssh',
                 if 'Archive' not in url:
                     source_url = url
                     break
+        if isinstance(source_url, list):
```

Comment:
This shoulde really check if it is a non-string instance of `collections.abc.Sequence`, rather than strictly being a list.